### ISSUE TITLE
Feature request: add "quantity" option and "state" of each amiibo (boxed, opened)

### ISSUE THREAD

**Issue by nolimit777** [EdwynZN/amiibo_network#67]
Hello !

Just wanted to share a feature request: would it be possible to add an option in settings to active "quantity" for each amiibo ?

I use to always buy 2 of each amiibo (sometimes 3 for my favorites) and I always keep 1 sealed in its mint box (well, that's a collector thing, I guess you already know how we are LOL), but I started to do this very lately in my quest of amiibo collection so I easily lost track of which I have already purchased more than once. I guess you see problem I have, there is so much amiibo now (I don't display all of them) that I forget which one I have already boxed or opened.

So this is the explanation and "justification" to add this option, I would REALLY enjoy to set quantity for each amiibo, and since everyone is not like me, I would suggest to add this option as "opt-in", I mean disabled by default but can be easily enabled via settings.

One more thing: I would scream of joy if we could have also the possibility to add the "state" of each amiibo in quantity (I mean: 2 boxed and 1 opened by example)

That"s all, do you think you could add these features easily please ?

Oh, almost forgot: thanks so much for the actually work, your app is a pure GEM and I just love you for this one.

Best Regards.

**Comment by EdwynZN** [comment]
Hello and thank you for your kindness. There features you asked would be really good, I'll be working on that later this month

`One more thing: I would scream of joy if we could have also the possibility to add the "state" of each amiibo in quantity (I mean: 2 boxed and 1 opened by example)`

That sounds like a fantastic idea and really important for collectors. I'll surely add it

**Comment by nolimit777** [comment]
Thanks for your fast answer and comment.

I'm so glad to read you and so happy to see that these features are interesting you too, to be honest I was really afraid to ask for them since I was truly convinced that they were going to be too personal and too specific requests to be considered and implemented.

So thank you very much for considering implementing them in future, can't wait to see them in action but don't worry, I'm not in a hurry ! As you know, collectors had to be (very) patient in their hobby ^^

Take care and thank for your openness, one last thing: I tried a lot of amiibo related app (or tools of the "poor": like excel sheets) and I can affirm your is definitively league ahead (UI, options, import/export...), I really wish it stays that way forever !

So, thank you for your time working on it.

Best Regards.

**Comment by EdwynZN** [comment]
Hello again with a quick question about this feature, does it makes sense for a box(or pack) for cards? i understand you don't really know what's in there until opened so just to be sure...

**Comment by nolimit777** [comment]
Hello !

Sorry for the very late reply, I hope it's not too late.
I don't collect amiibo cards personal but, indeed, no, it does not make really sense for cards to have the count of "opened" and "boxed" since each pack is a random set of cards.
But saying that, it's not impossible that some people are keeping and collecting some series unopened once their set is complete.

At best, you could add the count of unopened pack for each series but that mean you will have to add an new item in the app for each series.

By example, from the animal crossing ["nookipedia"](https://nookipedia.com/wiki/Amiibo#Welcome_amiibo_series) (😹😹😹) it means you will have to add these items to be able to count them as unopened/boxed (and "opened" state will not have very much sense here, so, "unopened/boxed" will be the only state possible from my point of view)

> Series 1
> Series 2
> Series 3
> Series 4
> Series 5
> Welcome amiibo series
> Sanrio series
> Promotional cards

But that probably mean also that you will have to add a special settings for these items to actually display them (something like "Display cards series package"), it may be confusing for some people and some hassle for you to add a feature I'm unsure people will use ¯\_(ツ)_/¯

**Comment by EdwynZN** [comment]
<img src="https://github.com/EdwynZN/amiibo_network/assets/16828084/d01adb66-e349-4188-8b9b-ddcc787373fa" width="250" height="500">
<img src="https://github.com/EdwynZN/amiibo_network/assets/16828084/87f467fd-8404-49bf-9f3c-2ffc0a0ee9c2" width="250" height="500">
<img src="https://github.com/EdwynZN/amiibo_network/assets/16828084/b7fe61dc-e82f-4722-87da-dc3dd7975271" width="250" height="500">
<img src="https://github.com/EdwynZN/amiibo_network/assets/16828084/cd124068-7c42-447c-be96-9b2b4285eaf6" width="250" height="500">

**Comment by nolimit777** [comment]
Hello my dear ^^

Thanks for the screenshots 😻, that looks absolutely perfect, I love so much your tiny logos representing boxed and unboxed: they are just so cute !
Everything looks very nicely implemented since you apparently take also account of global categories ! I can't wait to test it, thanks for your precious time 💖
The only thing I would suggest is *maybe* to put "Boxed" and "Unboxed" on a same line below "Owned" and "Wished" (so eveything on the right of the total count box).
But, honestly, this is only a small suggestion and probably belongs to personal preference ^^
And for the global switch to activate/deactivate this feature, this is simply exactly what I had in mind, thank you. It will avoid cluttering the app with a feature that some people have probably no need at all.

Again, thank you for all your past and new work about this app, it's really going to keep track of everything with precision for collectors.

Note: I wanted to ask if there is hidden a way to force language inside the app ? Should I open a new feature request ticket for this, very low priority, inquiry ?

Thanks for everything, 
Take care of you 💖💖💖

**Comment by EdwynZN** [comment]
> Note: I wanted to ask if there is hidden a way to force language inside the app ? Should I open a new feature request ticket for this, very low priority, inquiry ?

There is no way for now, the only 3 languages (English, Spanish and French} are based on the Android system, but thats something that could change ina future, just like you can select for light, dark and system theme

**Comment by EdwynZN** [comment]
The feature is online and will continue to improve it in the future
